File: CS/ViewModel/MainViewModel.cs
using DevExpress.Mvvm;
using DevExpress.Mvvm.DataAnnotations;
using DevExpress.Mvvm.POCO;
using System;
using System.Threading;

namespace SplashScreenService.ViewModel {
    [POCOViewModel]
    public class MainViewModel {
        public virtual int Delay { get; set; }

        protected ISplashScreenManagerService SplashScreenManagerService { get { return this.GetService<ISplashScreenManagerService>();} }

        public MainViewModel() {
            Delay = 5;
        }

        public void Display() {
            SplashScreenManagerService.ViewModel.Subtitle = "Powered by DevExpress";
            SplashScreenManagerService.ViewModel.Logo = new Uri("../../Images/Logo.png", UriKind.Relative);
            SplashScreenManagerService.Show();
            Thread.Sleep(TimeSpan.FromSeconds(Delay));
            SplashScreenManagerService.Close();
        }
    }
}

File: VB/ViewModel/MainViewModel.vb
Imports DevExpress.Mvvm
Imports DevExpress.Mvvm.DataAnnotations
Imports DevExpress.Mvvm.POCO
Imports System
Imports System.Threading

Namespace SplashScreenService.ViewModel
	<POCOViewModel>
	Public Class MainViewModel
		Public Overridable Property Delay() As Integer

		Protected ReadOnly Property SplashScreenManagerService() As ISplashScreenManagerService
			Get
				Return Me.GetService(Of ISplashScreenManagerService)()
			End Get
		End Property

		Public Sub New()
			Delay = 5
		End Sub

		Public Sub Display()
			SplashScreenManagerService.ViewModel = New DXSplashScreenViewModel()
			SplashScreenManagerService.ViewModel.Subtitle = "Powered by DevExpress"
			SplashScreenManagerService.ViewModel.Logo = New Uri("../../Images/Logo.png", UriKind.Relative)
			SplashScreenManagerService.Show()
			Thread.Sleep(TimeSpan.FromSeconds(Delay))
			SplashScreenManagerService.Close()
		End Sub
	End Class
End Namespace
